apps/data-scanner/lib/models/sessions/Session.mjs

```diff
@@ -3,37 +3,51 @@ import ModelBase from '../ModelBase.mjs';
 export class Session extends ModelBase {
     /**
      * @typedef {Class} Session
-     * @property readByPhone
+     * @property sessionId
      * @property readAll
      * @property save
      * @property update
      */
 
     /**
      * @method
-     * @param {String} phone - Phone nubmer that activates the session
+     * @param {String} sessionId - session idetifier that equel phone number that activates the session
      * @returns {Promise<Object>}
      */
-    async readByPhone({ phone }) {
-        return this.repository.session.readByPhone({ phone });
+    async readBySessionId({ sessionId }) {
+        return this.repository.session.readBySessionId({ sessionId });
     }
 
     /**
      * @method
+     * @param {String} sessionId - session idetifier that equel phone number that activates the session
+     * @param {Boolean} valid - if session is valid
+     * @param {Boolean} full - if session is full of phone numbers
```

Comment:
full --> isFull change in code everywere

Reply:
done